BUILDING.md

```diff
@@ -0,0 +1,87 @@
+# Build-Steps
+
+The J2V8 build-system performs several build steps in a fixed order to produce the final J2V8 packages for usage on the designated target platforms. What follows is a short summary for what each of the executed build-steps does and what output artifacts are produced by each step.
+
+---
+## Node.js
+
+Builds the [Node.js](https://nodejs.org/en/) & [V8](https://developers.google.com/v8/) dependency artifacts that are later linked against by the J2V8 native bridge code.
+
+__Inputs:__
+- Node.js source code
+    - see [Github](https://github.com/nodejs/node)
+- Node.js GIT patches with customizations for integrating Node.js into J2V8
```

Comment:
good to know, so building with another NodeJs is not out of the box. I didn't know one needed such patches (although I believe there was a legitimate reason).
👍